travis-scripts/jbossws-test.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -ev
+
+
+# uncomment to debug on non zero exit; be sure to print the correct log file
+#function finish {
+#  if [[ $USE_WFLY_MASTER = "true" ]]; then
+#     WFLY_TARGET=`pwd`"/wfly/wildfly/dist/target/"
+#     WFLY_HOME=$(find $WFLY_TARGET -name \wildfly\* -type d -maxdepth 1 -print | head -n1)
+#     echo "############ ERROR  dumping server.log #####################"
+#     cat $WFLY_HOME"/standalone/log/server.log"
+#  fi
+#}
+#
+#trap finish EXIT
+
+
+SERVER_VERSION=$1
+SECURITY_MGR=$2
+USE_WFLY_MASTER=$3
+
+D_SERVER_HOME=""
+
+if [[ $USE_WFLY_MASTER = "true" ]]; then
+
+  MYPWD=`pwd`
+
+  rm -rf wfly
+  mkdir wfly
+  cd wfly
+  git clone https://github.com/wildfly/wildfly.git
+  cd wildfly
+
+  # compile in silence.  The bld output is too much for travis log
+  mvn --quiet clean install -DskipTests
+
+  WFLY_TARGET=`pwd`"/dist/target/"
+  WFLY_HOME=$(find $WFLY_TARGET -name \wildfly\* -type d -maxdepth 1 -print | head -n1)
```

Comment:
Switch the params order to get rid of warning:
WFLY_HOME=$(find $WFLY_TARGET -maxdepth 1 -name \wildfly\* -type d -print | head -n1)